pkg/api/mlflow/dao/repositories/run.go

```diff
@@ -237,3 +250,37 @@ func (r RunRepository) SetRunTagsBatch(ctx context.Context, run *models.Run, bat
 	}
 	return nil
 }
+
+// getMinRowNum will find the lowest row_num for the slice of runs
+// or 0 for an empty slice
+func getMinRowNum(runs []models.Run) models.RowNum {
+	// get the lowest row_num in the slice
+	minRowNum := models.RowNum(0)
```

Comment:
the comment is redundant, and I _think_ it's more readable to just declare the variable like this?
```suggestion
	var minRowNum models.RowNum
```

Reply:
done